### DISCUSSION TITLE
A question about source maps

### DISCUSSION THREAD

**Discussion by kubk** [gradejs/gradejs#9] [Q&A] [answered]
Does this library only work with web sites that ship source maps to production? 

Since gradejs.com is built using [Webpack](https://github.com/fingerprintjs/gradejs/blob/master/webpack.config.js) I tried to make gradejs.com test itself. It says "It looks like the entered website is not built with Webpack": https://gradejs.com/w/gradejs.com

This website doesn't have public source maps. Are you going to provide a way to analyze bundles with no public source maps?

Public source maps are already information leak. That's why, for example, Sentry shows how to debug production app with private source maps: https://blog.sentry.io/2015/10/29/debuggable-javascript-with-source-maps#private-source-maps

**Comment by spalt08** [comment, marked as answer]
Hi Egor,

The current implementation doesn't rely on source maps at all. The gradejs.com is protected by Netlify password right now, so the application is facing Netlify utility page instead of real website with JS files. 

This can also happen with websites protected with tools like Cloudflare. We are currently thinking about a good way to handle such use cases.

**Reply by kubk** [reply]
Thanks for the answer 👍
